### PR TITLE
Add switches (on/off) for geniushub outlets

### PR DESCRIPTION
**Pull request by castaway**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#28182

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by zxdavb — DISMISSED** [review]
I think ready to go.

File: source/_integrations/geniushub.markdown [hunk 1]
```diff
@@ -7,24 +7,32 @@ ha_category:
...
-The `geniushub` integration links Home Assistant with your Genius Hub CH/DHW system, including its zones, devices, and issues.
+The `geniushub` integration links Home Assistant with your Genius Hub CH/DHW system, including its Zones, Devices, and Issues.
 
-It uses the [geniushub](https://pypi.org/project/geniushub-client/) client library, which provides data compatible with the v1 API that _may not_ necessarily match that of the official Web App.
+It uses the [geniushub](https://pypi.org/project/geniushub-client/) client library, which provides data compatible with the v1 API that _may not_ exactly match that of the official Web App.
 
 ### Zones
 
-Each zone controlled by your Genius Hub will be exposed as either a:
+Each Zone controlled by your Genius Hub will be exposed as either a:
```
Comment (frenck) [inline source/_integrations/geniushub.markdown]:
```suggestion
Each Zone controlled by your Genius Hub will be exposed as either a:
```

@zxdavb Why capitalize "Zone"? It is not a brand, but a regular word, that is not appearing in the beginning of a sentence, so it should be lower case.

Please adjust those occurrences.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/geniushub.markdown [hunk 2]
```diff
@@ -148,9 +165,9 @@ This is the recommended option.
 - Requires your **username** & **password**, as used with [geniushub.co.uk/app](https://www.geniushub.co.uk/app).
 - Uses the v3 API - unofficial, but there are additional features (e.g., battery levels).
 - Polls the hub directly (so is faster, say ~1s response time).
-- You have the option of specifying a MAC address.
+- You have the option of specifying a MAC address (not recommended, see above).
 
-The hub does not have to be in the same subnet as HA.
+The hub does not have to be in the same subnet as your HA server.
```
Comment (frenck) [inline source/_integrations/geniushub.markdown]:
```suggestion
The hub does not have to be in the same subnet as your Home Assistant server.
```

File: source/_integrations/geniushub.markdown (hunk 1 above)
Comment (zxdavb) [reply source/_integrations/geniushub.markdown]: done

File: source/_integrations/geniushub.markdown (hunk 2 above)
Comment (zxdavb) [reply source/_integrations/geniushub.markdown]: done

**Comment by zxdavb** [comment]
Ready for review.

**Review by frenck — APPROVED** [review]
LGTM! Thanks, guys! 👍

**Pull request merged**